test/grpc/route_guide/route-guide-eventuals-test.h

```diff
@@ -0,0 +1,75 @@
+#pragma once
+
+#include <gtest/gtest.h>
+
+#include <string>
+#include <vector>
+
+#include "eventuals/do-all.h"
+#include "eventuals/foreach.h"
+#include "eventuals/grpc/client.h"
+#include "eventuals/range.h"
+#include "test/grpc/route_guide/route-guide-eventuals-server.h"
+#include "test/grpc/route_guide/route_guide.eventuals.h"
+
+using eventuals::grpc::Client;
+using eventuals::grpc::ClientCompletionThreadPool;
+using routeguide::Feature;
+using routeguide::Point;
+using routeguide::Rectangle;
+using stout::Borrowable;
+
+using eventuals::grpc::Stream;
+using grpc::Status;
+
+using eventuals::DoAll;
+using eventuals::Finally;
+using eventuals::Foreach;
+using eventuals::Head;
+using eventuals::Let;
+using eventuals::Range;
+
+////////////////////////////////////////////////////////////////////////
+
+class RouteGuideTest : public ::testing::Test {
+ public:
+  RouteGuideTest() = default;
+
+  int GetPointsCount();
+
+  std::vector<Feature> feature_list_;
```

Comment:
Let's make this be a constant by parsing the file and setting this field in the constructor.